src/climate_data/data.py

```diff
@@ -154,15 +154,26 @@ def load_scenario_inclusion_metadata(self) -> pd.DataFrame:
     def daily_results_path(self, scenario: str, variable: str, year: int | str) -> Path:
         return self.daily_results / scenario / variable / f"{year}.nc"
 
+    def daily_results_path_with_draws(
+        self, scenario: str, variable: str, draw: int | str, year: int | str
+    ) -> Path:
+        return self.daily_results / scenario / variable / f"{year}_{draw}.nc"
+
     def save_daily_results(
         self,
         results_ds: xr.Dataset,
         scenario: str,
         variable: str,
+        draw: int | str | None,
         year: int | str,
+        provenance_attribute: str | None,
         encoding_kwargs: dict[str, Any],
     ) -> None:
-        path = self.daily_results_path(scenario, variable, year)
+        if draw is None:
```

Comment:
Is there a better way to do this?  Feels clunky.

Reply:
Shove it into `daily_results_path`.  E.g.

```python
    def daily_results_path(self, scenario: str, variable: str, year: int | str, draw: int | str | None = None) -> Path:
        file_name = f"{year}_{draw}.nc" if draw is not None else f"{year}.nc"
        return self.daily_results / scenario / variable / file_name
```
Then ditch your new `daily_results_path_with_draws` and pass the argument through.

Reply:
Thanks.